README.md

```diff
@@ -261,73 +259,350 @@ The SDK provides a simple interface for all the [media APIs mentioned here](http
 
-```cs
-PurgeCacheStatusResponse resp = await imagekit.GetPurgeCacheStatusAsync("cache_request_id");
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("FileId");
+List<String> tags = new ArrayList<>();
+tags.add("tag-to-remove-1");
+tags.add("tag-to-remove-2");
+ResponseMetaData resultTags=ImageKit.removeTags(new TagsRequest(fileIds, tags));
+
+```
+
+**8. Remove AI tags**
+
+Accepts an object of class `AITagsRequest` specifying the parameters to be used to remove AI tags. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/remove-aitags-bulk) can be passed via their setter functions to get the results.
+
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("629f3de17eb0fe4053615450");
+List<String> aiTags = new ArrayList<>();
+aiTags.add("Rectangle");
+AITagsRequest aiTagsRequest =new AITagsRequest();
+aiTagsRequest.FileIds=fileIds;
+aiTagsRequest.AITags=aiTags;
+ResponseMetaData resultTags = ImageKit.getInstance().removeAITags(aiTagsRequest);
+
+```
+
+**9. Delete File**
+
+Accepts the file ID and delete a file as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file).
+
+```java
+String fileId="your-file-id";
+ResponseMetaData result=ImageKit.deleteFile(fileId);
+
+```
+
+**10. Delete FileVersion**
+
+Accepts an object of class `DeleteFileVersionRequest` specifying the parameters to be used to delete file version. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/delete-file-version) can be passed via their setter functions to get the results.
+
+```java
+DeleteFileVersionRequest deleteFileVersionRequest = new DeleteFileVersionRequest();
+deleteFileVersionRequest.FileId="629d95278482ba129fd17c97";
+deleteFileVersionRequest.VersionId="629d953ebd24e8ceca911a66";
+ResponseMetaData resultNoContent = ImageKit.deleteFileVersion(deleteFileVersionRequest);
+
+```
+
+**11. Delete files (bulk)**
+
+Accepts the file IDs to delete files as per the [API documentation here](https://docs.imagekit.io/api-reference/media-api/delete-files-bulk).
+
+```java
+List<String> fileIds = new ArrayList<>();
+fileIds.add("your-file-id");
+fileIds.add("your-file-id");
+fileIds.add("your-file-id");
+
+ResponseMetaData result=ImageKit.bulkDeleteFiles(fileIds);
+
+```
+
+**12. Copy file**
+
+Accepts an object of class `CopyFileRequest` specifying the parameters to be used to copy file. All parameters specified in the [documentation here](https://docs.imagekit.io/api-reference/media-api/copy-file) can be passed via their setter functions to get the results.
+
+```java
+CopyFileRequest copyFileRequest = new CopyFileRequest();
+copyFileRequest.SourceFilePath="/w2_image.png";
+copyFileRequest.DestinationPath="/Gallery/";
+copyFileRequest.IncludeFileVersions=true;
```

Comment:
In Java, we have set `true` however, in Node.js readme, we have set `false`. The reason for this is that, in general, in code snippets, I am inclined towards mentioning this option in docs but with default value as mentioned in the API docs. So for this one, it would be `false`. To make things simple for you, just pick from Node.js readme.